hack/jenkins/job-configs/global.yaml

```diff
@@ -85,6 +85,33 @@
                     <healthScaleFactor>100.0</healthScaleFactor>
                 </hudson.tasks.junit.JUnitResultArchiver>
 
+- builder:
+    name: docker-build-publish
+    builders:
+        - raw:
+            xml: |
+               <com.cloudbees.dockerpublish.DockerBuilder plugin="docker-build-publish@1.1">
+                   <server plugin="docker-commons@1.2"/>
+                   <registry plugin="docker-commons@1.2">
+                       <credentialsId>---</credentialsId>
```

Comment:
How would the variable get populated with the value we want when the jjb sync is run?